bin/compile

```diff
@@ -18,7 +35,17 @@ cd $BUILD_DIR
 if [ -f shard.yml ]; then
   echo "-----> Installing Dependencies"
   $CRYSTAL_DIR/bin/crystal deps
+
+  if [ -f app.cr ]; then
```

Comment:
@zamith btw, if you don't mind breaking backwards compatibility, I'm happy to remove this - people should be using ``shard.yml``.